declarations/Reddit.json

```
@@ -8,8 +8,8 @@
     },
     "Privacy Policy": {
       "fetch": "https://www.redditinc.com/policies/privacy-policy",
-      "remove": ".filter-holder",
-      "select": "#content"
+      "remove": ".filter-holder, #overflow-x-scroll w-100",
```

`.filter-holder` does no target nothing, we could remove it.

I'm not sure to understand why are you removing ` #overflow-x-scroll w-100` ? 

If you think it's should be deleted, i think your seletor is wrong, i suggest
```suggestion
"remove": ".overflow-x-scroll.w-100"
```

but otherwise
```suggestion
    "Privacy Policy": {
      "fetch": "https://www.redditinc.com/policies/privacy-policy",
      "select": "#main-content"
    },
```